fabric-extensibility-api-v1/src/main/java/net/fabricmc/fabric/api/extensibility/item/v1/bow/BowInterface.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.extensibility.item.v1.bow;
+
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.projectile.PersistentProjectileEntity;
+import net.minecraft.item.ItemStack;
+
+/**
+ * An interface to implement for all custom bows in fabric. <br>
+ * Note: This is meant to be used on a BowItem class, the functionality will not work otherwise.
+ */
+public interface BowInterface {
```

Comment:
From the perspective of the API, the javadoc of the interface should explain all require things including some examples if needed.

I would `@see SimpleBowItem` at the bottom of class javadoc so people get linked to it.

Reply:
added to `BowInterface`, `CrossbowInterface`, and `TridentInterface`